src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package com.example.onlinebookstore.dto;
 
+import com.example.onlinebookstore.validation.UniqueIsbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
+    @UniqueIsbn
```

Comment:
use ready-to-use aannotations from jakarta.validation.*
```suggestion
    @Isbn
```